katalog/fluentd/README.md

```diff
@@ -6,14 +6,14 @@ Logging is configured to write records into Elasticsearch.
 
 ## Requirements
 
-- Kubernetes >= `1.10.0`
-- Kustomize >= `v1`
+- Kubernetes >= `1.14.0`
+- Kustomize >= `v3`
 - [prometheus-operator](https://github.com/sighup-io/fury-kubernetes-monitoring/blob/master/prometheus-operator)
 
 
 ## Image repository and tag
 
-* Fluentd image: `sighup/fluentd-elasticsearch:1.2.8`
+* Fluentd image: `quay.io/sighup/fluentd-elasticsearch:v1.10.2-debian-1.0`
```

Comment:
So in the end we went with our image, is there any reason for it?

Reply:
Yeah,
The main reason why we go with our own image is that the upstream one contains  fluentd configuration inside the docker image and the entrypoint does not allow to overwrite it. 

As we want to extend the configuration, is not easy to use the upstream one without a lot of hacking.

Thanks!